.github/workflows/tiobe.yml

```diff
@@ -0,0 +1,35 @@
+name: "TIOBE/TiCS"
+
+on:
+  schedule:
+    - cron: '0 0 * * TUE'
+
+jobs:
+  coverity:
+    if: github.repository == 'canonical/netplan'
+    runs-on: ubuntu-24.04
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          echo "APT::Get::Always-Include-Phased-Updates \"true\";" | sudo tee /etc/apt/apt.conf.d/90phased-updates
+          sudo sed -i 's/Types: deb/Types: deb deb-src/g' /etc/apt/sources.list.d/ubuntu.sources
+          sudo apt update
+          sudo apt build-dep -y netplan.io
+          sudo apt install -y gcovr python-is-python3 pylint flake8 python3-flake8
+      - name: Build Netplan
+        run: |
+          # The build directory must be called _build. TIOBE will look for coverage data in it.
+          meson setup _build --prefix=/usr -Db_coverage=true
+          meson compile -C _build -v
+          meson test -C _build -v
```

Comment:
@daniloegea Hmm we probably want to do something like this here (from `check-coverage.yml` workflow) to avoid this failure https://github.com/canonical/netplan/actions/runs/10361196598

```
 12       - name: Run unit tests
 11         run: |
 10           meson setup _build-cov --prefix=/usr -Db_coverage=true -Dunit_testing=true
  9           meson compile -C _build-cov
  8           unbuffer meson test -C _build-cov --verbose
```

Reply:
ack, I created another PR for that